### PR TITLE
445 fix and modify pca tools

### PR DESCRIPTION
**Pull request by nmaarnio**

### CODE REVIEW

File: eis_toolkit/cli.py [hunk 1]
```diff
@@ -715,27 +715,34 @@ def compute_pca_raster_cli(
...
-    variances_ratios_dict = {}
-    for i, variance_ratio in enumerate(variance_ratios):
-        name = "PC " + str(i) + " explained variance"
-        variances_ratios_dict[name] = variance_ratio
-    json_str = json.dumps(variances_ratios_dict)
+    # variances_ratios_dict = {}
+    # for i, variance_ratio in enumerate(variance_ratios):
+    #     name = "PC " + str(i) + " explained variance"
+    #     variances_ratios_dict[name] = variance_ratio
+    # json_str = json.dumps(variances_ratios_dict)
```
Comment (msorvoja) [inline eis_toolkit/cli.py:735]: Will these be needed later?

File: eis_toolkit/cli.py [hunk 2]
```diff
@@ -771,13 +778,20 @@ def compute_pca_vector_cli(
...
-        name = "PC " + str(i) + " explained variance"
-        variances_ratios_dict[name] = variance_ratio
-    json_str = json.dumps(variances_ratios_dict)
-
-    pca_gdf.to_file(output_vector)
+    # variances_ratios_dict = {}
+    # for i, variance_ratio in enumerate(variance_ratios):
+    #     name = "PC " + str(i) + " explained variance"
+    #     variances_ratios_dict[name] = variance_ratio
+    # json_str = json.dumps(variances_ratios_dict)
```
Comment (msorvoja) [inline eis_toolkit/cli.py:785]: Same as above

File: eis_toolkit/cli.py [hunk 3]
```diff
@@ -748,7 +755,7 @@ def compute_pca_raster_cli(
 def compute_pca_vector_cli(
     input_vector: INPUT_FILE_OPTION,
```
Comment (msorvoja) [inline eis_toolkit/cli.py:756]: Can INPUT_FILES_ARGUMENT be used here as well? Just for consistency.
```suggestion
    input_vector: INPUT_FILES_ARGUMENT,
```

**Review by msorvoja — APPROVED** [review]
Looks good! Left just a couple of comments.

File: eis_toolkit/cli.py (hunk 1 above)
Comment (nmaarnio) [reply eis_toolkit/cli.py:735]: Potentially yes

Comment (nmaarnio) [reply eis_toolkit/cli.py:735]: I prefer to keep them there

File: eis_toolkit/cli.py (hunk 3 above)
Comment (nmaarnio) [reply eis_toolkit/cli.py:756]: `INPUT_FILES_ARGUMENT` is used only when multiple datasets (rasters) are needed

**Pull request merged**